stonk_market_mayhem.py

```
@@ -0,0 +1,76 @@
+
+stocks = {
+    "AAPL": 150,
+    "GOOG": 2800,
+    "TSLA": 720,
+    "AMZN": 3400
+}
+
+portfolio = {}
+money = 10000
+
+def show_menu():
```

Ruff (0.8.2)

Missing return type annotation for public function `show_menu`

Add return type annotation: `None`

(ANN201)